.circleci/continue_config.yml

```diff
@@ -21,6 +21,7 @@ workflows:
   dist-compile:
     jobs:
       - linux-build
+      - native-specific-macos
```

Comment:
Should we move this line after "conditionals" below ?

Reply:
Yes, we don't have any tests for macOS build. It should be conditional.